composer.json

```diff
@@ -27,7 +27,7 @@
     "oat-sa/oatbox-extension-installer": "~1.1||dev-master",
     "ext-json": "*",
     "oat-sa/generis": ">=15.22",
-    "oat-sa/tao-core": ">=50.29",
+    "oat-sa/tao-core": ">=53.7.1",
```

Comment:
- [ ] Just a small reminder to be sure this will be the final version as someone can release tao-core mean while :)

Reply:
I will check ofc

Reply:
https://github.com/oat-sa/tao-core/releases/tag/v53.7.1